### PR TITLE
:sparkles: Add customized error pages (404, 403, 5**)

### PR DESCRIPTION
**Pull request by ElodieBichet**
To test error pages in dev environment:
{root_url}/_error/404
{root_url}/_error/403
{root_url}/_error/500 (or any status code)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +0,0 @@
-###> symfony/framework-bundle ###
```
Comment (XenoX) [inline .env:1]: Why deleting this file ?

File: templates/bundles/TwigBundle/Exception/error.html.twig [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Erreur 404 -
```
Comment (XenoX) [inline templates/bundles/TwigBundle/Exception/error.html.twig:3]: We can set the title block on one line

File: templates/bundles/TwigBundle/Exception/error.html.twig [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	{{ parent() }}
+{% endblock %}
+
+{% block header %}
+	<header class="masthead" style="background-image: url({{ asset('img/error.jpg') }})">
+		<div class="container position-relative px-4 px-lg-5">
+			<div class="row gx-4 gx-lg-5 justify-content-center">
+				<div class="col-md-10 col-lg-8 col-xl-7">
+					<div class="post-heading">
+						<h1>Erreur
```
Comment (XenoX) [inline templates/bundles/TwigBundle/Exception/error.html.twig:13]: We can set the h1 on one line

File: templates/bundles/TwigBundle/Exception/error.html.twig [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+			</div>
+		</div>
+	</header>
+{% endblock %}
+
+{% block body %}
+	<main class="mb-4">
+		<div class="container px-3 px-lg-4">
+			<p>
+				Désolé, la page demandée ne fonctionne pas.<br>
```
Comment (XenoX) [inline templates/bundles/TwigBundle/Exception/error.html.twig:27]: We can write two <p>, one per ligne and remove the <br> :)

File: .env (hunk 1 above)
Comment (ElodieBichet) [reply .env:1]: Please ignore this file in the pull request, I just slipped up.

**Pull request merged**